src/assets/Generator.Shared/ResponseHeadersExtensions.cs

```diff
@@ -12,6 +12,34 @@ namespace Azure.Core
 {
     internal static class ResponseHeadersExtensions
     {
+        private static readonly string[] KnownFormats =
```

Comment:
It looks like we have some redundancy with formats in multiple places, including https://github.com/Azure/autorest.csharp/blob/feature/v3/src/assets/Generator.Shared/TypeFormatters.cs#L14.  Would it make sense to centralize, or is there a reason to keep DateTime formatting separately?